pkg/scripts/uninstall.sh

```diff
@@ -121,7 +122,7 @@ destroy_tfstate() {
     if [ ${S3_BUCKET_EXISTS} -eq 0 ]
     then
       set +e
-      terraform destroy -auto-approve "${OVERRIDE_CONFIG_FILE}"
+      terraform destroy -auto-approve "${OVERRIDE_CONFIG_FILE}" | tee -a "${LOG_FILE}"
```

Comment:
If you `tee` the log file, then the `$?` is the result of the `tee` command, which is usually `0` and not dependent on the `terraform destroy` result. If you want to check the outcome in the subsequent condition, you need to add `set -o pipefail` to the top of the script.